reference/conanfile/tools/google/bazeltoolchain.rst

```diff
@@ -0,0 +1,130 @@
+.. _conan_tools_google_bazeltoolchain:
+
+BazelToolchain
+==============
+
+.. important::
+
+    Some of the features used in this section are still **under development**, while they are
+    recommended and usable and we will try not to break them in future releases, some breaking
+    changes might still happen if necessary to prepare for the *Conan 2.0 release*.
+
+Available since: `1.37.0 <https://github.com/conan-io/conan/releases/tag/1.37.0>`_
+
+BazelToolchain
+--------------
+
+The ``BazelToolchain`` is the toolchain generator for Bazel. It will generate a ``conan_bzl.rc`` file that contains
+a build configuration ``conan-config`` to inject all the parameters into the :command:`bazel build` command.
+
+The ``BazelToolchain`` generator can be used by name in conanfiles:
+
+.. code-block:: python
+    :caption: conanfile.py
+
+    class Pkg(ConanFile):
+        generators = "BazelToolchain"
+
+.. code-block:: text
+    :caption: conanfile.txt
+
+    [generators]
+    BazelToolchain
+
+And it can also be fully instantiated in the conanfile ``generate()`` method:
+
+.. code-block:: python
+    :caption: conanfile.py
+
+    from conan import ConanFile
+    from conan.tools.google import BazelToolchain
+
+    class App(ConanFile):
+        settings = "os", "arch", "compiler", "build_type"
+
+        def generate(self):
+            tc = BazelToolchain(self)
+            tc.generate()
+
+After running :command:`conan install conanfile.py` command, the ``BazelToolchain`` generates the *conan_bzl.rc* file
+that contains Bazel build parameters (it will depend on your current Conan settings and options from your *default* profile):
+
+.. code-block:: text
+    :caption: conan_bzl.rc
+
+    # Automatic bazelrc file created by Conan
+
+    build:conan-config --cxxopt=-std=gnu++17
+
+    build:conan-config --dynamic_mode=off
+    build:conan-config --compilation_mode=opt
+
+The :ref:`Bazel build helper<conan_tools_google_bazel>` will use that ``conan_bzl.rc`` file to perform a call using this
+configuration. The outcoming command will look like this :command:`bazel --bazelrc=/path/to/conan_bzl.rc build --config=conan-config <target>`.
+
+
+The toolchain supports the following methods and attributes:
+
+constructor
++++++++++++
+
+.. code-block:: python
+
+    def __init__(self, conanfile, namespace=None):
+
+
+- ``conanfile``: the current recipe object. Always use ``self``.
+- ``namespace``: (deprecated since Conan 1.62) this argument avoids collisions when you have multiple toolchain calls in the same
+  recipe. By setting this argument, the *conanbuild.conf* file used to pass information to the
+  build helper will be named as: *<namespace>_conanbuild.conf*. The default value is ``None`` meaning that
+  the name of the generated file is *conanbuild.conf*. This namespace must be also set with the same
+  value in the constructor of the ``Bazel`` build helper so that it reads the information from the proper
+  file.
```

Comment:
If this is deprecated, and more than that, it's not even taken into account from the 1.62 version, I would remove the explanations about the parameter and just say that it's deprecated, that it will have no effect